Keas.Mvc/Controllers/TagsController.cs

```diff
@@ -11,7 +11,6 @@
 
 namespace Keas.Mvc.Controllers
 {
-    [Authorize(Policy = "DepartmentAdminAccess")]
```

Comment:
I think it's better to move this back up to cover the whole controller, and then add [AllowAnonymous] on the listTags action. That way if we later change the policy to something else (say TagMaster), we only have to change in one place.

Reply:
@laholstege Well I don't want to make it truly [AllowAnonymous] because only department users should be able to access this, but I also agree that we should leave the authorize policy as-in on the main controller in here.  Probably the way to go is just to move the listTags into a new controller, or maybe into the assetcontrolelr